src/cbc_binary_toolkit/ubs.py

```diff
@@ -213,12 +213,12 @@ def download_hashes(cbth, hashes, expiration_seconds=3600):
     return found_hashes
 
 
-def get_metadata(cbth, binary):
+def get_metadata(cbc_api, binary):
     """
     Initiates download of binary metadata from Unified Binary Store.
 
     Args:
-        cbth (cbapi.CbThreatHunterAPI): Carbon Black ThreatHunter object.
+        cbc_api (cbc_sdk.CBCloudAPI): Carbon Black ThreatHunter object.
```

Comment:
We'll wanna change this like the others to "Carbon Black Cloud API object"

Reply:
Thanks! I thought I caught most but still have a little more to do